.github/workflows/utility/update_chaindata.py

```diff
@@ -0,0 +1,26 @@
+import requests
+import json
+import os
+from os import getcwd
+from os.path import isfile, isdir, join
+
+rootdir = getcwd()
+
+#Gets a list of all the json file locations for all chains (sorts out unnecessary files)
+def jsonlist():
+    for chainfolder in os.listdir(rootdir):
+        chainjson = os.path.join(chainfolder, "chain.json")
+        if os.path.isfile(chainjson):
+            return(chainjson)
+        
+chainjsonlist = jsonlist()
+
+def checkUpdate(jsonlist):
+     for chainjson in jsonlist:
+         # test case plz remove later
+        if chainjson == "osmosis/chainjson":
```

Comment:
We need to update the Osmosis chain.json to get this working / test it right?

Reply:
I don't think so, since the current implementation of the chain.json on our end is just for the "codespace" part of the chain.json schema on the chian-registry. I'll share a working run soon.

Reply:
Sorry unclear answer on my part - the osmosis chain.json has already been updated, no further action req here.